### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by nagaraju-vadakoppula**
updated readme based on >> https://competent-bartik-ba3ce3.netlify.app/iteration/1/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,62 @@
...
+- testing framework and 
+- browser automation technology
+
+> **_NOTES:_**
+>- This project is currently in `Draft` state. If you do not find any relevant example (for Deque supported languages), contact [Deque HelpDesk](mailto:helpdesk@deque.com).
+>- You need a valid license to access and use our APIs and example projects. For more information, check out [our website](https://deque.com).
+
+The supported languages are as follows:
+
+### C#/.NET
```
Comment (michael-siek) [inline README.md:16]: This is an accessibility violation https://dequeuniversity.com/rules/axe/4.4/heading-order?application=RuleDescription

File: README.md [hunk 2]
```diff
@@ -1,9 +1,62 @@
...
+
+- MS Test
+- NUnit
+
+### JAVA
```
Comment (michael-siek) [inline README.md:21]:
```suggestion
### Java
```

**Review by michael-siek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nagaraju-vadakoppula) [reply README.md]: Agreed. Skipped the higher heading levels to keep the heading text size to be appealing.

As a workaround, bolded the heading text.

**Review by michael-siek — CHANGES_REQUESTED** [review]
small nitpick

**Pull request merged**